include/vcpkg/base/messages.h

```diff
@@ -1681,6 +1736,20 @@ namespace vcpkg
                     "",
                     "dependency {spec} was expected to be at least version "
                     "{expected_version}, but is currently {actual_version}.");
+    DECLARE_MESSAGE(VersionDateHelp, (), "", "A date (2021-01-01.5)");
+    DECLARE_MESSAGE(VersionFourFlavors, (), "", "Versions in vcpkg come in four primary flavors");
```

Comment:
Should we change this to something like "the following versions are accepted" or similar so that the message need not be retranslated if/when the set changes?